packages/host/src/lib/csi-controller.ts

```diff
@@ -556,6 +556,10 @@ export class CSIController extends TypedEmitter<Events> {
             this.createInstanceAPIRouter();
 
             this.bpmux = new BPMux(streams[8]);
+            this.bpmux.on("error", (e: any) => {
+                this.logger.warn("Instance client multiplex connection errored", e.message);
+                streams[8]?.end();
```

Comment:
the error was thrown then hub client send request to kill Instance, right? why do we need to end the hub client stream in such case (and only this one)?

Reply:
This also ends all the requests sent over the connections. This will clean up the connections and get the allocated memory back.

Reply:
clear, thanks